roles/ncn-common/files/scripts/metal/metal-lib.sh

```diff
@@ -186,6 +186,16 @@ function configure_lldp() {
     echo 'LLDP is configured for all bond interfaces and their members (mgmt and sun)'
 }
 
+# Return netmask for a given netmask [0-32].
+cidr_to_netmask() {
+    local cidr
+    local value
+
+    cidr="$(echo $1 | awk -F/ '{print $NF}')"
+    value=$(( 0xffffffff ^ ((1 << (32 - $cidr)) - 1) ))
```

Comment:
whoa hope this works

Reply:
This needs a rebase, it dragged in a commit from something else.

Reply:
It does work though!